### ISSUE TITLE
Update six requirement

### ISSUE THREAD

**Issue by href** [PyFilesystem/pyfilesystem2#79]
pyfilesystem2 requires six version `1.10.*` in `setup.py`:

    "six~=1.10.0"

Is this a hard requirement? Version 1.11.0 of six has just been released and we've been hitting the following error ever since, when setting up our testing infrastructure:

```python
Traceback (most recent call last):
  File ".tox/py34/lib/python3.4/site-packages/_pytest/vendored_packages/pluggy.py", line 510, in load_setuptools_entrypoints
    plugin = ep.load()
  File ".tox/py34/lib/python3.4/site-packages/pkg_resources/__init__.py", line 2404, in load
    self.require(*args, **kwargs)
  File ".tox/py34/lib/python3.4/site-packages/pkg_resources/__init__.py", line 2427, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File ".tox/py34/lib/python3.4/site-packages/pkg_resources/__init__.py", line 872, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (six 1.11.0 (.tox/py34/lib/python3.4/site-packages), Requirement.parse('six~=1.10.0'), {'fs'})
```

I'm not able to currently pinpoint the issue (to provide you with a test), but I can fix the problem by pinning six to `1.10.0`. In some sense this is a pip issue, but since pyfilesystem2 limits six to `1.10.*` it's only a matter of time until it is incompatible with some other package that'll require `1.11.0+`.

Could the requirement maybe be changed to `"six>=1.10.0"`?

**Comment by willmcgugan** [comment]
Yes, that should be fine. I will release an alpha version soon with that change.

**Comment by althonos** [comment]
I'd be more in favor of using `six ~= 1.10` so that `six==2` is not installed should there be a release one day.

**Comment by willmcgugan** [comment]
@href I've released `fs==2.0.10a1` if you would like to try that. six version is set as per @althonos's suggestion.

**Comment by href** [comment]
That fixes it, thanks!
